Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfMeshPrimitiveAttributes.cs

```diff
@@ -2,28 +2,114 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 
 namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
 {
     /// <summary>
     /// Common mesh primitive attributes.
     /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/mesh.primitive.schema.json
     /// </summary>
-    /// <remarks>
-    /// Application specific semantics are not supported
-    /// </remarks>
-    [Serializable]
-    public class GltfMeshPrimitiveAttributes
+    public class GltfMeshPrimitiveAttributes : Dictionary<string, int>
```

Comment:
Nice change! From your other question, I think I'm good with this changing to a `ReadOnlyDictionary`. We can add a note to the changelist about it, but I'm not sure it makes sense for these to need updating after deserialization.

Reply:
Also, if the worry is people messing with the dictionary when it's returned, could this instead be implemented with a private dictionary in the class instead of the class being a dictionary itself? The constructor could still take one in and save it locally, and `TryGetDefault`'s `TryGetValue` would be called on the private dictionary instead of itself.